src/ui/layout/mct-tree.vue

```diff
@@ -430,9 +432,40 @@ export default {
 
             return scrollTopAmount >= treeStart && scrollTopAmount < treeEnd;
         },
+        sortNameDescending(a, b) {
+            // sorting tree children items
+            if (!(a.name && b.name)) {
+                if (a.object.name > b.object.name) {
+                    return 1;
+                }
+
+                if (b.object.name > a.object.name) {
+                    return -1;
+                }
+            }
+
+            // sorting compositon items
+            if (a.name > b.name) {
+                return 1;
+            }
+
+            if (b.name > a.name) {
+                return -1;
+            }
+
+            return 0;
+        },
```

Comment:
@akhenry, @jvigliotta - May I get your input on the sorting algorithm? This naive sorting implementation might cause problems if we expected mixed alphanumeric characters in object names. Such cases as the attached photo might arise with numerics failing to sort as expected.

![Screen Shot 2021-11-01 at 9 50 47 AM](https://user-images.githubusercontent.com/7417521/139691423-4b1cdf80-e0d5-4656-873b-b4f11598c06a.png)

If we expect alphanumeric sorting to sort more naturally, I can adjust the algorithm.

Reply:
I think we should just use string sorting rather than attempting to do some combination of string and numerical sorting. At least for a V1.

Reply:
Perfect, thanks Andrew!